ossapi/ossapiv2.py

```diff
@@ -554,6 +554,14 @@ def ranking(self,
         return self._get(Rankings, f"/rankings/{mode.value}/{type_.value}",
             params=params)
 
+    @request
+    def spotlights(self) -> List[Spotlight]:
+        """
+        https://osu.ppy.sh/docs/index.html#get-spotlights
+        This method returns a list of `Spotlight`s rather than a `Spotlights` model,
+        but since that's all the model contains it is functionally equivalent. 
```

Comment:
eh I don't think this is necessary, at the most I would leave an inline comment above `self._get` (but even then probably not), docstrings are intended to be useful for consumers more than for developers. Consumers don't care why it's a list of spotlights, they just want to know the return type.